### PR TITLE
Update tests

### PR DESCRIPTION
**Pull request by JSAssassin**
This PR: 

- Updates the test to check for cryptosuite type. New spec updates requires the `cryptosuite` type value to be either `eddsa-rdfc-2022` or `eddsa-jcs-2022`.

- Adds a test to check if dereferencing the "verificationMethod" results in an object containing a type property with "Multikey" value.

- Additionally, removes unused `generate-credentials` script.
- Adds test to check if "publicKeyMultibase" is 35 bytes in length and is multibase base58-btc encoded.
- Adds test to check if "proof.proofPurpose" field matches the verification relationship expressed by the verification method controller.

### CODE REVIEW

File: tests/10-create.js
Comment (JSAssassin) [inline tests/10-create.js:127]: I'm not sure if the assertion here is correct. The spec states that the byte length of the `publicKeyMultibase` should be 35 (see - https://www.w3.org/TR/vc-di-eddsa/#multikey). However, when the `publicKeyMulibase` is decoded, it appears to have a byte length of 34. Here's an example of what the publicKeyMultibase looks like: `z6Mkq6FxLL9UT6uLihkHRrjqnz7K9TDHhbTTH3bvJ7BbUgup`.

Is it possible that that is an error in the spec. What I noticed is that in the spec, it mentions a two-byte prefix (0xed01), which, when I encoded using varint, results in three bytes. Also, the prefix of "ed25519" in the multicodec table is 0xed (See -https://github.com/multiformats/multicodec/blob/master/table.csv#L97).

It would be great if someone could confirm if this is an error.

Comment (dlongley) [reply tests/10-create.js:127]: @JSAssassin,

Someone requested that the varint text be deleted in this spec (and not the other, apparently), and this has led to some confusion. `0xed` as a varint is `0xed01`. So what's in the spec is already expressed as a varint -- so you don't need to convert it again. You can put just `0xed` into the varint API to produce the right value: `0xed01`, which is the value the spec says (it just doesn't mention varint anymore).

Comment (dlongley) [reply tests/10-create.js:127]: Looks like the 35 is a copy-paste error from the ECDSA spec which has 35 byte key representations (as there's an extra "compression header" of a single byte there). It should be 34, so the tests should say that and we'll have to correct the typo in the spec.

Comment (JSAssassin) [reply tests/10-create.js]: @dlongley Thanks a lot, I've fixed in the test as well as opened a PR to fix it in the spec https://github.com/w3c/vc-di-eddsa/pull/67

**Review by aljones15 — APPROVED** [review]
LGTM please do wait on review from @BigBlueHat

File: CHANGELOG.md
Comment (TallTed) [inline CHANGELOG.md:6]:
```suggestion
- Add test to check whether dereferencing the `verificationMethod` results in an
```

File: CHANGELOG.md
Comment (TallTed) [inline CHANGELOG.md:8]:
```suggestion
- Add test to check whether `publicKeyMultibase` is 34 bytes in length and is
```

File: CHANGELOG.md
Comment (TallTed) [inline CHANGELOG.md:10]:
```suggestion
- Adds test to check whether `proof.proofPurpose` field matches the verification
```

**Review by TallTed — CHANGES_REQUESTED** [review]
editorial, human-facing tweaks

**Review by TallTed — APPROVED** [review]

**Review by dlongley — APPROVED** [review]

**Pull request merged**
